resources/lang/en/admin/hardware/table.php

```diff
@@ -6,6 +6,7 @@
     'asset_model'       => 'Model',
     'book_value'  	=> 'Current Value',
     'change' 		=> 'In/Out',
+    'checkin_date' => 'Checkin Date',
```

Comment:
Do we want to say "Checkin Date" here or re-use "Last checkin date"? We're not providing a log of all checkouts an item has ever had, so it could maybe be confusing?

Reply:
Changed it to `Last Checkin Date` 👍🏾